lightning/src/ln/chan_utils.rs

```diff
@@ -197,7 +200,9 @@ impl CounterpartyCommitmentSecrets {
 		res
 	}
 
-	pub(crate) fn provide_secret(&mut self, idx: u64, secret: [u8; 32]) -> Result<(), ()> {
+	/// Inserts the secret `secret` at index `idx`. Returns Ok(()) if the secret
```

Comment:
I think you can leave off the leading non-ticked words "secret" and "index". Likewise elsewhere.

Reply:
Done I hope I got them all